virttest/qemu_storage.py

```
@@ -1511,7 +1550,16 @@ def info(self, force_share=False, output="human"):
                 # tls creds objects of the backing images
                 cmd += " %s" % " ".join(self._backing_access_tls_creds_objects)
 
-            if "--backing-chain" in self.help_text:
+            if self.qemu_ver in VersionInterval("(,10.1.0)"):
+                help_text = self.help_text
+            else:
+                help_text = process.run(
+                    self.image_cmd + f" info -h",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)